frontends/precinct-scanner/src/screens/admin_screen.tsx

```diff
@@ -37,6 +38,10 @@ interface Props {
   usbDrive: UsbDrive;
 }
 
+const BallotsScannedText = styled.div`
+  font-size: larger;
```

Comment:
wow I've never seen `larger` before! cool!

I've seen in the `ui` library in the `Prose` component that there's a `theme` prop to control the font size. I'm not totally sure how to use it but that might be another way to go about this that would use standardized font sizes

Reply:
cc @beausmith

Reply:
Good to know about `Prose`! I'll wait for Beau's input since I don't believe this is urgent to merge.